wordpress-mobile/gutenberg-mobile
# Fix sizeSlug does not update in Gallery block

### Related PR:

`gutenberg`: https://github.com/WordPress/gutenberg/pull/27691

To test (test on web and mobile):

* Create a new post
* Add a gallery block
* Add a few images
* Tap block settings
* Change image size setting (try "Medium")

Expect the size setting to be reflected after selecting the option.

PR submission checklist:
